core/src/main/java/com/datastax/oss/driver/internal/core/session/DefaultSession.java

```diff
@@ -74,14 +79,8 @@
 public class DefaultSession implements CqlSession {
 
   private static final Logger LOG = LoggerFactory.getLogger(DefaultSession.class);
-
-  public static CompletionStage<CqlSession> init(
-      InternalDriverContext context,
-      Set<InetSocketAddress> contactPoints,
-      CqlIdentifier keyspace,
-      Set<NodeStateListener> nodeStateListeners) {
-    return new DefaultSession(context, contactPoints, nodeStateListeners).init(keyspace);
-  }
+  private static final Logger STARTUP_LOG =
+      LoggerFactory.getLogger("com.datastax.oss.driver.api.core.Startup");
```

Comment:
Not sure if we need a dedicated log, the message can already be disabled through the configuration if it's unwanted.

Reply:
My idea was that users might want to attach special appenders to this logger and/or switch off its additivity to give these messages a special treatment (think monitoring tools that are only interested in knowing how many sessions were successfully created, and when).